server/data/adjudicationsApiClient.test.ts

```diff
@@ -3,7 +3,7 @@ import config from '../config'
 import { adjudicationSummaryWithActiveMock } from './localMockData/miniSummaryMock'
 import AdjudicationsApiRestClient from './adjudicationsApiClient'
 
-jest.mock('./tokenStore')
+jest.mock('./cache/tokenStore')
```

Comment:
A lot of the change in this PR is related to the move of `TokenStore` into the new `data/cache` folder